docs/cudf/source/developer_guide/testing.md

```diff
@@ -0,0 +1,143 @@
+# Testing cuDF
+
+## Tooling
+Tests in cuDF are written using [`pytest`](https://docs.pytest.org/en/latest/).
+Test coverage is measured using [`coverage.py`](https://coverage.readthedocs.io/en/6.4.1/),
+specifically the [`pytest-cov`](https://github.com/pytest-dev/pytest-cov) plugin.
+Code coverage reports are uploaded to [Codecov](https://app.codecov.io/gh/rapidsai/cudf).
+Each PR also indicates whether it increases or decreases in test coverage.
+
+## Test organization
+
+How tests are organized depends on which of the following two groups they fall into:
+
+1. Free functions operating on the above classes like `cudf.merge`.
```

Comment:
“Above” should be “below?” Or swap the order.

Reply:
Ah yeah I swapped the order because it made the discussion below flow better but forgot to reword this.